build.gradle

```diff
@@ -494,6 +494,7 @@ project('segmentstore:server:host') {
         compile project(':segmentstore:server')
         compile group: 'ch.qos.logback', name: 'logback-classic', version: qosLogbackVersion
         testCompile project(':test:testcommon')
+        testCompile project(path:':common', configuration:'testRuntime')
```

Comment:
Why is this needed?

Reply:
Since the new class(`TestSimpleCache`) is needed (covered in the next comment), a test time dependency would have to be introduced.